clang/lib/Parse/ParseCXXInlineMethods.cpp

```diff
@@ -124,7 +124,7 @@ NamedDecl *Parser::ParseCXXInlineMethodDef(
                       ? diag::warn_cxx98_compat_defaulted_deleted_function
                       : diag::ext_defaulted_deleted_function)
         << 0 /* defaulted */;
-      Actions.SetDeclDefaulted(FnD, KWLoc);
+      Actions.SetDeclDefaulted(nullptr, FnD, KWLoc);
```

Comment:
```suggestion
      Actions.SetDeclDefaulted(/*Scope=*/nullptr, FnD, KWLoc);
```

Ditto everywhere you pass a nullptr.

Alternatively, declare `SetDeclDefaulted` as

``void SetDeclDefaulted(Decl *dcl, SourceLocation DefaultLoc, Scope *S = nullptr);`

Reply:
Got it, I will go with `/*Scope=*/`.